client/server/pages/index.js

```diff
@@ -352,6 +352,16 @@ function getDefaultContext( request, entrypoint = 'entry-main' ) {
 		context.commitChecksum = getCurrentCommitShortChecksum();
 	}
 
+	if ( calypsoEnv === 'jetpack-cloud-stage' ) {
+		context.badge = 'jetpack-cloud-staging';
+		context.feedbackURL = 'https://github.com/Automattic/wp-calypso/issues/';
```

Comment:
I wonder if https://github.com/Automattic/wp-calypso/issues/new?labels=Jetpack%20Cloud would even make more sense.

Reply:
Thought about it, and I disagree. For one, the other links are all to the plain issues, not the new issue link, so let's keep that. For another, given the crossover between JP Cloud and Calypso, it'll be harder for someone not familiar with the codebase to know what's one or the other. Better let us triage and label appropriately.